Content.Trauma.Server/Chaplain/HolyFlammableSystem.cs

```diff
@@ -0,0 +1,503 @@
+using Content.Goobstation.Shared.Religion;
+using Content.Server._Goobstation.Wizard.Systems;
+using Content.Server.Administration.Logs;
+using Content.Server.Stunnable;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Alert;
+using Content.Shared.Atmos;
+using Content.Shared.Atmos.Components;
+using Content.Shared.Damage.Systems;
+using Content.Shared.Database;
+using Content.Shared.Interaction;
+using Content.Shared.Inventory;
+using Content.Shared.Physics;
+using Content.Shared.Popups;
+using Content.Shared.Projectiles;
+using Content.Shared.Rejuvenate;
+using Content.Shared.Temperature;
+using Content.Shared.Throwing;
+using Content.Shared.Timing;
+using Content.Shared.Toggleable;
+using Content.Shared.Weapons.Melee.Events;
+using Content.Trauma.Shared.Chaplain;
+using Content.Trauma.Shared.Chaplain.Components;
+using Microsoft.EntityFrameworkCore.Storage;
+using Robust.Server.Audio;
+using Robust.Shared.Configuration;
+using Robust.Shared.Physics.Components;
+using Robust.Shared.Physics.Events;
+using Robust.Shared.Physics.Systems;
+using Robust.Shared.Random;
+
+
+
+/// <summary>
+/// Adds an action ability that will cause all flammable targets in a radius to ignite, also heals the owner
+/// of the component when used.
+/// </summary>
+namespace Content.Trauma.Server.Chaplain
```

Comment:
remove extra newlines above it, update the docs to not be copy paste (and for the system not the namespace)

file scoped namespace